config/fcos/v1_6_exp/validate_test.go

```diff
@@ -234,3 +236,110 @@ func TestValidateGrubUser(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateConfig(t *testing.T) {
+	tests := []struct {
+		in      Config
+		out     error
+		errPath path.ContextPath
+	}{
+		// valid config (wipe_table is true)
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Disks: []base.Disk{
+							{
+								Device:    "/dev/vda",
+								WipeTable: util.BoolToPtr(true),
+								Partitions: []base.Partition{
+									{
+										Label: util.StrToPtr("foo"),
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		// valid config (disk is /dev/disk/by-id/coreos-boot-disk)
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Disks: []base.Disk{
+							{
+								Device:    rootDevice,
+								WipeTable: util.BoolToPtr(false),
+								Partitions: []base.Partition{
+									{
+										Label:  util.StrToPtr("bar"),
+										Number: 1,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		// invalid config (wipe_table is nil)
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Disks: []base.Disk{
+							{
+								Device:    "/dev/vda",
+								WipeTable: nil,
+								Partitions: []base.Partition{
+									{
+										Label: util.StrToPtr("foo"),
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			out:     common.ErrReuseByLabel,
+			errPath: path.New("yaml", "storage", "disks", 0, "partitions", 0, "number"),
+		},
+		// invalid config (wipe_table is false with a partition numbered 0)
```

Comment:
Did you mean to have an explicit `WipeTable: util.BoolToPtr(false)`?

Reply:
I did, i thought that it was false by default but it makes more sense to keep it in thanks!

Reply:
Yeah, `bool` is false by default, but `*bool` is nil by default, and we want to check how we behave with `&false`.  Fix looks correct, thanks.